frontend/src/bundles/common/components/header-wrapper/components/resumes-header/resumes-header.tsx

```diff
@@ -0,0 +1,59 @@
+import { useCallback } from 'react';
+import { renderToString } from 'react-dom/server';
+
+import shareIcon from '~/assets/img/share-icon.svg';
```

Comment:
consider import svg as react component
```suggestion
import ShareIcon from '~/assets/img/share-icon.svg?react';
```

Reply:
Done